conans/client/migrations_settings.py

```diff
@@ -2403,7 +2403,9 @@
         libcxx: [libstdc++, libc++]
         cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20]
     intel:
-        version: ["11", "12", "13", "14", "15", "16", "17", "18", "19", "19.1"]
+        version: ["11", "12", "13", "14", "15", "16", "17", "18", "19", "19.1",
+                  "2021.1", "2021.2", "2021.3"]
+            update = [None, ANY]
```

Comment:
spacing

Reply:
why switch to year instead of compiler version?

Reply:
Apparently, it's the version https://software.intel.com/content/www/us/en/develop/articles/oneapi-toolkit-version-to-compiler-version-mapping.html

Reply:
really? can you double-check what is returned for `__INTEL_COMPILER` and `__INTEL_COMPILER_UPDATE`?
if you have an access to the compiler, ofcourse.
curious, cause it may break our detection.